loadtest/control/simulcontroller/actions.go

```diff
@@ -2016,6 +2039,22 @@ func (c *SimulController) openUserProfile(u user.User) control.UserActionRespons
 	if err := u.GetChannelMember(ch.Id, post.UserId); err != nil {
 		return control.UserActionResponse{Err: control.NewUserError(err)}
 	}
+
+	cpaEnabled, resp := control.CustomProfileAttributesEnabled(u)
+	if resp.Err != nil {
+		return resp
+	}
+	if cpaEnabled {
+		// attempt to retrieve from store
+		attributes := u.Store().GetCPAValues(post.UserId)
+		if len(attributes) == 0 {
```

Comment:
Is this logic similar to the webapp where we try to fetch from redux store, and if not there, we make an API call?